index.html

```diff
@@ -20,377 +20,273 @@
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="fourth-tags">
-          <div class="fourth-tag">
-           <span>Html</span>
-          </div>
-          <div class="fourth-tag2">
-           <span>bootstrap</span>
+      <div class="card-works">
+        <div class="feature">
+          <div class="img-placeholder">
+            <img src="assets/img-placeholder.png" alt=""/>
           </div>
-          <div class="fourth-tag3">
-           <span>Ruby</span>
-          </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="fifth-tags">
-          <div class="fifth-tag">
-           <span>Html</span>
-          </div>
-          <div class="fifth-tag2">
-           <span>bootstrap</span>
-          </div>
-          <div class="fifth-tag3">
-           <span>Ruby</span>
+          <div class="right-block">
+            <h1>Multi-Post Stories</h1>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a standard dummy text.
+            </p>
+            <div class="tag">
+              <ul>
+                <li>Css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+            <div class="block-btn">
+              <h4>See Project</h4>
+            </div>
           </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
         </div>
 
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="sixth-tags">
-          <div class="sixth-tag">
-           <span>Html</span>
+        <div class="card-1">
+          <div class="first-card">
+            <div class="card-cursor"></div>
+            <div class="card-action">
+              <div class="action-btn">
+                <h4>See Project</h4>
+              </div>
+            </div>
           </div>
-          <div class="sixth-tag2">
-           <span>bootstrap</span>
+          <div class="second-card">
+            <div class="second-block">
+              <h1>Data Dashboard Healthcare</h1>
```

Comment:
- [ ] [Optional] I would suggest you please use `hx` tag here like `h2, h3,h4` because only we use the `h1` tag per page and it represents the main heading of the page.